src/SeisSol.cpp

```diff
@@ -116,16 +117,20 @@ bool seissol::SeisSol::init(int argc, char* argv[]) {
     const auto rlimInKb = rlim.rlim_cur / 1024;
     // Softlimit (rlim_cur) is enforced by the kernel.
     // This limit is pretty arbitrarily set at 2GB.
-    const auto reasonableStackLimit = 2097152; // [kb]
+    constexpr auto reasonableStackLimitInKb = 0x200'000ULL;                    // [kiB]
```

Comment:
can you precise in the comment how much it is? (2Gb?)

Reply:
Yup, it's 2 GiB, I'm gonna add a comment